docker/Dockerfile

```diff
@@ -51,10 +51,12 @@ ENV \
 COPY /dist/requirements-frozen.txt /tmp/requirements-frozen.txt
 RUN set -x \
   && buildDeps="" \
-  # uwsgi
+  # uwsgi-dogstatsd
   && buildDeps="$buildDeps \
   gcc \
+  libpcre3-dev \
   wget \
+  zlib1g-dev \
```

Comment:
pyuwsgi is prebuilt with pcre and zlib support -- any plugin that gets built needs to also have access to those headers since they're part of `uwsgi.h`